hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson

```diff
@@ -0,0 +1,80 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the primary cfg hjson for DV linting. It imports ALL individual lint
+  // cfgs of the IPs DV environments and the full chip DV environment for top_earlgrey.
+  // This enables to run them all as a regression in one shot.
+  name: top_earlgrey_dv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/data/{tool}.hjson"]
+
+  // Different dashboard output path for each tool
+  rel_path: "hw/top_earlgrey/dv_lint/{tool}"
```

Comment:
```suggestion
  rel_path: "hw/top_earlgrey/dv/lint/{tool}"
```
The reasoning here is - we should have only 4 'flow' specific directories - {dv, syn, lint, fpv}. We keep the directory structure before the 'flow' consistent with the location of the fusesoc core file being used to run it. So we would have `<ip>/lint` and `<ip>/dv/lint` areas and from the directory structure, we can quickly find the fusesoc core file. 

In that vein, perhaps move this file to `hw/top_earlgrey/dv`?